lib/metadataTypes/AccountUser.js

```diff
@@ -32,7 +55,7 @@ class AccountUser extends MetadataType {
             await this.client.soap.retrieveBulk(
                 'AccountUserAccount',
                 ['AccountUser.AccountUserID', 'AccountUser.UserID', 'Account.ID', 'Account.Name'],
-                { options: optionsBUs }
+                optionsBUs
```

Comment:
im going to assume this is intentional :) just wanted to check

Reply:
yes, your function is wrapping that directly. otherwise it would have been options.options.something